### ISSUE TITLE
[INFO] Room Signage

### ISSUE THREAD

**Issue by ngjoh** [nexi-intra/nexi-meeting#18]
### TLDR;

[ ] Integrated with db in xxxx

I'll create a prototype of a mobile assistant that gets loaded when scanning the QR code, and adjust the room signage to be non-interactive. The interactions will be moved to the mobile UI.

First, let's update the room signage to be non-interactive:

```typescriptreact project="CAVA" file="nexi-room-signage-static.tsx"
...
```

Now, let's create the mobile assistant UI that would be loaded when scanning the QR code:

```typescriptreact project="CAVA" file="nexi-mobile-assistant.tsx"
...
```

Now, to demonstrate how these components would work together in a real-world scenario, let's create a page that simulates both the room signage and the mobile assistant side by side:

```typescriptreact project="CAVA" file="nexi-room-management-demo.tsx"
...
```

This demo page showcases both the static room signage and the interactive mobile assistant side by side, simulating how they would work together in a real-world scenario. The room signage remains non-interactive and displays the QR code, while the mobile assistant provides all the interactive features for managing the room status and bookings.

Key features of this prototype:

1. The room signage is now static, only displaying the current status, time, and QR code.
2. The mobile assistant, accessed by scanning the QR code, provides all interactive features:

1. Viewing current room status
2. Quick booking functionality
3. Ability to change room status
4. Viewing room features and capacity

3. Both components maintain Nexi Group branding with appropriate colors and logos.
4. The mobile assistant is designed to be user-friendly on smaller screens, with easy-to-tap buttons and clear status indicators.

This setup allows for a more realistic simulation of how the room management system would work in practice, with the physical signage being updated based on actions taken through the mobile interface.
